scripts/run_experiment.py

```diff
@@ -110,10 +111,20 @@ def run_experiment(argv):
         help='Pickled data for variant configuration')
     parser.add_argument(
         '--use_cloudpickle', type=ast.literal_eval, default=False)
+    parser.add_argument(
+        '--save_in_gcs_bucket',
```

Comment:
we want to also be able to save it to a prefix path in the bucket. 

this is achievable by having the user provide a full gcs path rather than just the bucket name. e.g.
`gs://my-cool-bucket/the/path/just/keeps/going`

Reply:
That's done by setting this [field](https://github.com/rlworkgroup/garage/blob/004c05f18b849bd6740952be69ced24a0f08ca11/garage/config_personal_template.py#L53) in the config_personal file. Should I pass it as a parameter of the function run_experiment instead?